test/solidity/Facets/AllBridgeFacet.t.sol

```diff
@@ -168,4 +205,114 @@ contract AllBridgeFacetTest is TestBaseFacet {
+
+        vm.expectRevert(InvalidCallData.selector);
+
+        allBridgeFacet.startBridgeTokensViaAllBridge(
+            bridgeData,
+            validAllBridgeData
+        );
+    }
+
+    function testRevert_WhenReceiverDoesNotMatch() public {
+        vm.startPrank(USER_SENDER);
+
+        // update bridgeData
+        validAllBridgeData.recipient = bytes32(uint256(uint160(USER_SENDER)));
+
+        usdc.approve(address(allBridgeFacet), bridgeData.minAmount);
+
+        vm.expectRevert(InvalidReceiver.selector);
+
+        allBridgeFacet.startBridgeTokensViaAllBridge(
+            bridgeData,
+            validAllBridgeData
+        );
+    }
+
+    function testRevert_InvalidNonEVMReceiver() public {
+        vm.startPrank(USER_SENDER);
+
+        // update bridgeData
+        bridgeData.receiver = NON_EVM_ADDRESS;
+        validAllBridgeData.recipient = bytes32(0);
+
+        usdc.approve(address(allBridgeFacet), bridgeData.minAmount);
+
+        vm.expectRevert(InvalidNonEVMReceiver.selector);
+
+        allBridgeFacet.startBridgeTokensViaAllBridge(
+            bridgeData,
+            validAllBridgeData
+        );
+
+        vm.stopPrank();
+    }
+
+    function test_ChainIdMapping() public {
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_ETHEREUM),
+            ALLBRIDGE_ID_ETHEREUM
+        );
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_OPTIMISM),
+            ALLBRIDGE_ID_OPTIMISM
+        );
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_BSC),
+            ALLBRIDGE_ID_BSC
+        );
+        // tron
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_TRON),
+            ALLBRIDGE_ID_TRON
+        );
+        // solana
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_SOLANA),
+            ALLBRIDGE_ID_SOLANA
+        );
+        // polygon
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_POLYGON),
+            ALLBRIDGE_ID_POLYGON
+        );
+        // arbitrum
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_ARBITRUM),
+            ALLBRIDGE_ID_ARBITRUM
+        );
+        // avalanche
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_AVALANCHE),
+            ALLBRIDGE_ID_AVALANCHE
+        );
+        // base
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_BASE),
+            ALLBRIDGE_ID_BASE
+        );
+        // celo
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_CELO),
+            ALLBRIDGE_ID_CELO
+        );
+        // sui
+        assertEq(
+            allBridgeFacet.getAllBridgeChainId(LIFI_CHAIN_ID_SUI),
+            ALLBRIDGE_ID_SUI
+        );
+        // unknown
+        vm.expectRevert(UnsupportedAllBridgeChainId.selector);
+
+        allBridgeFacet.getAllBridgeChainId(1290);
+    }
```

Comment:
do we need demoAllBridge.ts script to test it?

Reply:
no, the facet is already integrated and working, this would be throwaway work